pkgs/applications/science/math/gap/default.nix

```diff
@@ -61,13 +63,23 @@ in
 stdenv.mkDerivation rec {
   pname = "gap";
   # https://www.gap-system.org/Releases/
-  version = "4.11.1";
+  version = "4.12.0";
 
   src = fetchurl {
     url = "https://github.com/gap-system/gap/releases/download/v${version}/gap-${version}.tar.gz";
-    sha256 = "sha256-ZjXF2n2CdV+DOUhrnKwzdm9YcS8pfoI0+6QIGJAuowQ=";
+    sha256 = "sha256-YAXQwDFv6IA9hGgfML5k6s6DqVQvl19cs97frdQ5GEk=";
   };
 
+  postPatch = ''
+    # https://github.com/gap-system/gap/issues/5015
+    # TODO: remove after updating
+    sed -i 's|body = fail|body = false|' pkg/atlasrep/gap/access.gi
+
+    # "make install" tries to install the GAP Development manual,
+    # which is not in the tarball (Git only)
+    sed -i 's|for book in ref tut hpc dev|for book in ref tut hpc|' Makefile.rules
```

Comment:
Ouch! BTW, if you have the time, I always appreciate an issue report for such things ;-) -- not necessary in this case, though, I've already [submitted a fix](https://github.com/gap-system/gap/pull/5091) which should be in 4.12.1. Thanks for the other one.

Reply:
I normally submit issues for such things, I was just feeling a bit embarrassed for reporting an already-reported problem before :)

Reply:
Actually, it was not clear to me that the issue you reported there was already reported; and in any case, it was not possible for you to know there was a fix IMHO. And even if: from my POV, you didn't do anything embarrassing!  I'd appreciate anyone taking the time to report a constructive issue report, even if it turns out to be solved already :-).